src/distill-components/pal-footer-template.js

```diff
@@ -56,7 +56,7 @@ export const palFooterTemplate = `
   <a href="/" class="logo">
     ${logo}
     Predictive Analytics Laboratory
-  </a> is dedicated to solving big problems with big ideas
+  </a> does machine learning research
```

Comment:
```suggestion
  </a> does innovative machine learning research
```

Reply:
I know it's a bit boring but let's go with this until Novi signs off on something else.